command/help.go

```diff
@@ -0,0 +1,45 @@
+package command
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	assets "vcListBot/command/assets"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+type HelpData struct {
+	Title       string `json:"title"`
+	Explanation string `json:"explanation"`
+}
+
+func Help(session *discordgo.Session, message *discordgo.MessageCreate) {
+	if message.Author.ID == session.State.User.ID {
+		return
+	}
+	if message.Content != "!help" {
+		return
+	}
+	data, err := ioutil.ReadFile("./command/assets/help.json")
+	if err != nil {
+		fmt.Println("error loading file,", err)
+	}
+	var helpData []HelpData
+	err = json.Unmarshal(data, &helpData)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
このままだとエラーが発生した後も処理が続行してしまいます
```suggestion
		fmt.Println(err)
		return
```